README.md

```diff
@@ -49,7 +49,7 @@ The following policy grants the user access to the bucket `my-restricted-bucket`
             "Sid": "AllowBucketStat",
             "Effect": "Allow",
             "Action": [
-                "s3:HeadBucket"
+                "s3:ListBucket"
             ],
             "Resource": "*"
```

Comment:
Actually, should this also be referencing `arn:aws:s3:::my-restricted-bucket`?

Reply:
I can't remember if it makes any difference to the way [MinIO](https://github.com/minio/mc) works, but my guess is that it's probably fine if you want to restrict the `ListBucket` action to a specific resource, too.